etc/kayobe/environments/ci-builder/stackhpc-ci.yml

```diff
@@ -25,6 +25,7 @@ kolla_enable_octavia: true
 kolla_enable_opensearch: true
 kolla_enable_prometheus: true
 kolla_enable_redis: true
+kolla_enable_skyline: true
```

Comment:
Let's decide as a group whether or not to support this.

Reply:
Removed